hypothesis-python/src/hypothesis/extra/pandas/impl.py

```diff
@@ -204,6 +206,7 @@ def indexes(
       should pass a max_size explicitly.
     * unique specifies whether all of the elements in the resulting index
       should be distinct.
+    * name is passed to the pandas.Index constructor.
```

Comment:
```suggestion
    * name is a strategy for strings or ``None``, which will be passed to the :class:`pandas.Index` constructor.
```

Cross-references in docs mean we don't have to spell out what exactly constitutes a valid name.  (and again below)